src/applications/disability-benefits/all-claims/tests/pages/bddRedirect.unit.spec.jsx

```diff
@@ -98,15 +108,17 @@ describe('526 bddRedirect page', () => {
 
   it('should show the right content when the separation date is > 180 days in the future', () => {
     const tree = mount(
-      <DefinitionTester
-        schema={schema}
-        uiSchema={uiSchema}
-        data={formData(
-          moment()
-            .add(181, 'days')
-            .format(dateFormat),
-        )}
-      />,
+      <Provider store={defaultStore}>
+        <DefinitionTester
+          schema={schema}
+          uiSchema={uiSchema}
+          data={formData(
+            moment()
+              .add(181, 'days')
+              .format(dateFormat),
+          )}
+        />
+      </Provider>,
```

Comment:
Now that the eBen link is a container, can this be restored to not use a `Provider`? Same case with `militaryHistory.unit.spec.jsx`